cpp/core/CMakeLists.txt

```diff
@@ -162,7 +168,7 @@ set(CODEGEN_HEADERS
     third_party/
     )
 set(THIRD_PARTY_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/third_party)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${THIRD_PARTY_INCLUDE})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${THIRD_PARTY_INCLUDE} ${VELOX_HOME})
```

Comment:
If we support other backend in the future, this include will be not suitable

Reply:
Thanks, I use `if(BUILD_VELOX_BACKEND)` to decide whether we need this.

Reply:
Can you also move it to velox CmakeLists.list